app/models/qc_asset_creator/multiple_tag2_conversion.rb

```diff
@@ -24,13 +41,24 @@ def asset_update_state
   end
 
   ##
-  # Transfers the source plate into the target plate
+  # Transfers the source plate into the target plate using bulk_transfers
+  # which allow for custom transfer templates. We only transfer wells
+  # contining tag2 tubes.
   # The child is ignored (Although it should be the same as the target plate)
   # Applies tags
   # Applies tag2s
   def asset_transfer(_)
 
-    super
```

Comment:
This is the main change. We use the bulk transfer to perform an entirely custom transfer.